hazelcast/src/main/java/com/hazelcast/config/PermissionConfig.java

```diff
@@ -199,6 +231,15 @@ public static PermissionType getType(String nodeName) {
             return null;
         }
 
+        public static PermissionType getTypeByPermissionClassName(String permissionClassname) {
```

Comment:
Where is this used?

Reply:
It's used in the hazelcast-enterprise. I've updated the PR description with more details and a link to the other PR.